src/usr/local/www/includes/functions.inc.php

```diff
@@ -33,12 +33,13 @@ function get_stats() {
 	$stats['states'] = get_pfstate();
 	$stats['temp'] = get_temp();
 	$stats['datetime'] = update_date_time();
-	$stats['interfacestatistics'] = get_interfacestats();
-	$stats['interfacestatus'] = get_interfacestatus();
+	$stats['interfacestatistics'] = 0;//get_interfacestats();//unused
+	$stats['interfacestatus'] = 0;//get_interfacestatus();//unused
```

Comment:
Removed now, including the functions they call, and changed the index numbers that changed by removing them from the array.